client/.eslintrc.js

```diff
@@ -20,9 +20,15 @@ module.exports = {
   rules: {
     // allow debugger during development
     "no-debugger": process.env.NODE_ENV === "production" ? 2 : 0,
-    "jsx-a11y/anchor-is-valid": "off",
+    "jsx-a11y/anchor-is-valid": [
+      "error",
```

Comment:
Do we not need `"components": [ "Link" ]`

Reply:
Actually not. I was trying to make it work in a way any anchor would be rejected. As the Link is more versatile, it can actually not receive hrefs. I wanted to enforce any Link with click handlers were set to role="button", but that doesn't seem to be possible as well.